src/protagonist/Engine/Ingest/Image/ThumbCreator.cs

```diff
@@ -31,8 +31,7 @@ public async Task<int> CreateNewThumbs(Asset asset, IReadOnlyList<ImageOnDisk> t
             logger.LogDebug("No thumbs to process for {AssetId}, aborting", assetId);
             return 0;
         }
-        
-        var expectedSizes = asset.GetAvailableThumbSizes(asset.FullThumbnailPolicy!, out var maxDimensions, true);
+        var expectedSizes = asset.GetAvailableThumbSizes(out var maxDimensions, true);

```

Comment:
We don't want to do this - use the sizes of the images as provided from Cantaloupe. Also see comment on `GetThumbnailSize()` in this class too - it states we use sizes we've calculated but with this change we want to use sizes returned from Cantaloupe